### PR TITLE
Adding NY animation

### PR DESCRIPTION
**Pull request by akurasov**

### CODE REVIEW

File: examples/visual-effects/src/main/kotlin/HappyNY.kt [hunk 1]
```diff
@@ -0,0 +1,365 @@
...
+    Box(modifier) {
+        snowFlakeInt(0, 0f, 30.dp, 0.dp, alpha)
+        snowFlakeInt(0, 60f, 15.dp, 25.dp, alpha)
+        snowFlakeInt(0, 120f, -15.dp, 25.dp, alpha)
+        snowFlakeInt(0, 180f, -30.dp, 0.dp, alpha)
+        snowFlakeInt(0, 240f, -15.dp, -25.dp, alpha)
+        snowFlakeInt(0, 300f, 15.dp, -25.dp, alpha)
+    }
+
+}
```
Comment (olonho) [inline examples/visual-effects/src/main/kotlin/HappyNY.kt:355]: Add 
```kotlin
tasks.register("runNewYear") {
        additionalArguments.add("NY")
        dependsOn(runTask)
    }
```
to build.gradle.kts

File: examples/visual-effects/src/main/kotlin/HappyNY.kt [hunk 2]
```diff
@@ -0,0 +1,365 @@
...
+    }
+    val colors = arrayOf(Color.Red, Color.Yellow, Color.Green, Color.Yellow, Color.Cyan, Color.Magenta, Color.White)
+    for (i in 0..starCount) {
+        stars.add(Star( (width*random()).dp, (height*random()).dp, colors[Random.nextInt(colors.size)], (3 + 5*random()).dp))
+    }
+}
+
+@OptIn(ExperimentalComposeUiApi::class)
+@Composable
+fun NYContent() {
```
Comment (olonho) [inline examples/visual-effects/src/main/kotlin/HappyNY.kt:200]: Add `@Preview` annotation.

File: examples/visual-effects/src/main/kotlin/HappyNY.kt [hunk 3]
```diff
@@ -0,0 +1,365 @@
...
+import kotlin.math.*
+import kotlin.random.Random
+
+val width = 1200
+val height = 800
+val snowCount = 80
+val starCount = 60
+val rocketPartsCount = 30
+
+data class SnowFlake(var x: Dp, var y: Dp, val scale: Float, var v: Double, var alpha: Float, var angle : Float, var rotate : Int, var phase : Double)
```
Comment (olonho) [inline examples/visual-effects/src/main/kotlin/HappyNY.kt:31]: Extra space after phase.

Comment (olonho) [reply examples/visual-effects/src/main/kotlin/HappyNY.kt:31]: Better reformat with IDEA tools.

File: examples/visual-effects/src/main/kotlin/HappyNY.kt (hunk 2 above)
Comment (akurasov) [reply examples/visual-effects/src/main/kotlin/HappyNY.kt:241]: Done

File: examples/visual-effects/src/main/kotlin/HappyNY.kt (hunk 1 above)
Comment (akurasov) [reply examples/visual-effects/src/main/kotlin/HappyNY.kt:419]: Done

File: examples/visual-effects/src/main/kotlin/HappyNY.kt (hunk 3 above)
Comment (akurasov) [reply examples/visual-effects/src/main/kotlin/HappyNY.kt]: Done

**Review by olonho — APPROVED** [review]

**Pull request merged**